crates/core/src/vm.rs

```diff
@@ -271,7 +272,7 @@ fn iter_by_col_range<'a>(
     ctx: &'a ExecutionContext,
     db: &'a RelationalDB,
     tx: &'a TxMode,
-    table: DbTable,
+    table: &'a DbTable,
```

Comment:
I'm not sure what's causing the regression in `full-join`, but this is extra indirection.

Reply:
It's unclear where this regression is coming from at the moment.